pkg/fetcher/fetcher.go

```diff
@@ -461,15 +461,20 @@ func extractErrResponseMessage(rdr io.ReadCloser) (string, error) {
 	res := []byte(out.Bytes())
 	log.Debugf("Error message json string: %s", string(res))
```

Comment:
I logged the error response json string in two places:
1. every time when there's an error response body (here)
2. when the err json is malformed
basically if the err json has expected format, the complete string will be logged once 
if the err json is irregular, the err string will be logged twice
I just got comments from @matthewavery that logging at debug level is expensive
is it better if I just log it only when the json is malformed, and log it at info level instead? 
(ping @sflxn for advice

Reply:
loggin at debug level is not expensive, we want to avoid logging anything that might be large above the `debug` level. The debug level is for when we are wanting the maximum amount of logging from VIC. This means that it will slow us down the most, we want to avoid logging anything above debug if we do not have to since that can cause performance or space issues. We want to log this above the debug level(in my opinion) because it is valuable for us to get the information on the first failure. if we would be logging the malformed json body twice(above debug) then that is likely not necessary.

Reply:
i see. yes you're definitely right that it's valuable to log the first failure (malformed json response) above debug level. fixing this right now.
about logging the complete response json for every error (not only the malformed one), i think we can do it at debug level - it's additional information to the client (the error json could contain other details not extracted to the command line output)